client/packages/system/src/Patient/PatientView/AddButton.tsx

```diff
@@ -36,6 +36,12 @@ export const AddButton = () => {
     SplitButtonOption<PatientModal>
   >(options[1] as SplitButtonOption<PatientModal>);
 
+  const onSelectOption = (option: SplitButtonOption<PatientModal>) => {
+    setSelectedOption(option);
```

Comment:
as requested.. I was finding the select and click action slightly irritating when testing.

Reply:
Yes feels good